src/common/commonutils/CommonUtils.h

```diff
@@ -4,6 +4,7 @@
 #ifndef COMMONUTILS_H
 #define COMMONUTILS_H
 
+#include <Logging.h>
```

Comment:
Aha! Now I remember. 

One reason we had the opaque handle as a void* was this: we do not want to link commonutils to the logging library. By including Logging.h we now do this lin. The idea is that commonutils can be linked only by itself, for example as it is for the UTs, and all those optional void* arguments are passed as null.